src/Command.php

```diff
@@ -200,23 +200,23 @@ public function setCommand(string $command): void
      */
     private function initTemplate(): void
     {
-        $this->setTemplate(static::TEMPLATE_COMMAND_NAME . static::TEMPLATE_OPTIONS . static::TEMPLATE_URL);
+        $this->setTemplate(self::TEMPLATE_COMMAND_NAME . self::TEMPLATE_OPTIONS . self::TEMPLATE_URL);
```

Comment:
Why do you think it is a good idea to change `static` to `self`?

Reply:
There are no reasons to use `static` in final class.